### DISCUSSION TITLE
Gen 2 B-hyve Smart Hose Watering Timer

### DISCUSSION THREAD

**Discussion by KJThaDon** [sebr/bhyve-home-assistant#248] [Q&A]
Will this likely support the newer generation model?
#21204
https://www.orbitonline.com/products/b-hyve-smart-1-outlet-watering-timer-gen-2?variant=45719879385256

**Comment by sebr** [comment]
Probably

**Reply by anseld1986** [reply]
Yes, it works with them. If you already have it setup you'll need to delete the integration and re-add it to get then to show up.
